extension/redirects.js

```diff
@@ -28,6 +28,15 @@ const KNOWN_REDIRECTS = [
     ],
     types: ["main_frame"]
   },
+  {
+    name: 'Facebook',
+    targetParam: 'u',
+    patterns: [
+        '*://l.facebook.com/l.php?',
```

Comment:
Very minor comment, but please use 2-space indents like the rest of the code. My editor is complaining that there's an indentation here of 8 characters when it should be just 6.

Reply:
Fixed 😉